synapse/lib/ast.py

```diff
@@ -2118,7 +2124,7 @@ async def run(self, runt, genr):
             async for (verb, iden) in node.iterEdgesN2():
                 wnode = await runt.snap.getNodeByBuid(s_common.uhex(iden))
                 if wnode is not None:
-                    yield wnode, path.fork(wnode)
+                    yield wnode, path.fork(wnode, link={'type': 'edge', 'edge': verb, 'reverse': True})
```

Comment:
Lets switch the `edge` key to `verb` since it's more consistent with the existing graph projection and seems to fit a bit better. ( here and any other `type=edge` links )